packages/utils/src/eslint-utils/RuleCreator.ts

```diff
@@ -13,10 +13,11 @@ export type NamedCreateRuleMetaDocs = Omit<RuleMetaDataDocs, 'url'>;
 
 export type NamedCreateRuleMeta<
   MessageIds extends string,
+  Options extends readonly unknown[],
```

Comment:
Adding this new generic in this location is a breaking change.
It changes the meaning of existing code and could introduce type errors.
Instead it must be added after all type parameters AND it must be given a default value of `[]`